qiskit/primitives/base/base_sampler.py

```diff
@@ -11,9 +11,66 @@
 # that they have been altered from the originals.
 
 r"""
-===================
-Overview of Sampler
-===================
+=====================
+Overview of SamplerV2
+=====================
+
+:class:`~BaseSamplerV2` is a primitive that samples bitstrings from quantum circuits.
+
+Following construction, a sampler is used by calling its :meth:`~.BaseSamplerV2.run` method
+with a list of pubs (Primitive Unified Blocks). Each pub contains two values that, together,
+define a computation unit of work for the sampler to complete:
+
+* a single :class:`~qiskit.circuit.QuantumCircuit`, possibly parameterized, whose final state we
+  define as :math:`\psi(\theta)`,
```

Comment:
It looks like we don't use the latex later, and also, don't want to emphasize final states for V2 sampler

```suggestion
* A single :class:`~qiskit.circuit.QuantumCircuit`, possibly parameterized.
```